canonical/jaas-dashboard
# WD-1958 - Applications search

## Done

- Implement applications search
- Selectable search result
- Run an action on a list of applications

## TODO

- This feature is only supported on local apps at the moment, as I'm not sure how remote apps work

## QA

- Run the project locally
- Go to the application list 
- Perform a search
- Make sure that the selection is working properly
- Click on run action, once you have selected some applications
- Make sure that the charm panel is shown properly
- Select  a charm to run actions on
- Continue the this scenario
- Make sure that the action is executed on the appropriate applications/units

## Details
## Screenshots

![image](https://user-images.githubusercontent.com/36013798/219065699-66f65aed-df76-4e01-b07a-ced439d46da3.png)
